ILSANG/Sources/Network/UserNetwork.swift

```diff
@@ -27,4 +28,10 @@ final class UserNetwork {
             return false
         }
     }
+    
+    func getUserRank(xpstat: String, page: Int) async -> Result<UserRank, Error> {
+        let parameters: Parameters = ["xpType":xpstat, "size": page]
```

Comment:
```suggestion
    func getUserRank(xpstat: String) async -> Result<UserRank, Error> {
        let parameters: Parameters = ["xpType":xpstat, "size": 20]
```

20개만 불러오도록 기능명세되어있어서, 고정해둬도 될 것 같습니다 !!